packages/fxa-content-server/tests/server/amplitude.js

```diff
@@ -294,6 +312,26 @@ registerSuite('amplitude', {
       });
     },
 
+    'screen.settings.change-password': () => {
+      amplitude(
+        {
+          ...getBasicEvent('screen.settings.change-password'),
+        },
+        {
+          ...BASIC_REQUEST,
+        },
+        {
+          ...BASIC_DATA,
+        }
+      );
```

Comment:
nit: this looks duplicated&hellip; a lot. But not sure how/if we can DRY it slightly.

```js
function createAmplitudeEvent(type='', request=BASIC_REQUEST, data=BASIC_DATA) {
  return amplitude(
    {....getBasicEvent(type)},
    {...request},
    {...data}
  );
}

amplitude(createAmplitudeEvent('screen.settings.change-password'));
```

I guess conversely we could convert the args into an object instead of positional to make it easier to pass just `type` and overridden `BASIC_DATA` values. 🤷 

I can't remember if I saw any instances below of custom `BASIC_REQUEST` or `BASIC_DATA` overrides. But using some sort of utility function might reduce each of these [34] blocks by ~9 lines o' code.

Reply:
👍 I felt that way about this entire file haha, we don't have anything DRY'd up  here. I tried something similar but not exactly ran into issues so I backed it up a step in DRYness, but I'll try it out, smaller diff is better diff. Thanks!